src/mame/ibm/thinkpad760.cpp

```diff
@@ -0,0 +1,110 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/*************************************************************************************************************
+Skeleton driver for IBM ThinkPad 760 series.
+IBM ThinkPad 760 was a notebook computer introduced in 1995 by IBM as part of the ThinkPad 700-series.
+Eleven models were produced: 760C, 760CD, 760L, 760LD, 760E, 760ED, 760EL, 760ELD, 760XL, 760XD, and 760D/L.
+
+Hardware for the 760XD model.
+  CPU PCB:
+    -Intel Mobile Pentium MMX 166 (TT80503166).
+    -Two Samsung KM732V589T-15 (Cache SRAM, 32KX32).
+    -One IDT 71V256 (Lower Power 3.3V CMOS Fast SRAM 256K [32K x 8-Bit]).
+  RAM PCB:
+    -Four on-board Toshiba TC51V18165CFTS-60 (1M X 16 EDO DRAM).
+    -Two DIMM slots.
+  Main PCB:
+    -Intel PCIset SB82437MX.
+    -Intel PCIset SB82371FB.
+    -Two Intel PCIset FA82438MX.
+    -W48C60-402G.
+    -Four M5M4V18165CTTP (1M x 16 EDO DRAM).
+    -Dallas DS17485S-5 (real-time clock/calendar).
+    -NEC 53G9037.
+    -NEC 53G9038.
+    -IBM 89G7219.
+    -IBM 20H2888.
+    -IBM 94G0207.
+    -HD6433436A18F (mask ROM, IBM-branded).
+    -Maxim MAX3243CAI.
+    -C46CM6 (SEEPROM, for BIOS settings).
+    -Texas Instruments PCIbus PCI1130PDV.
+    -Texas Instruments 90G9510.
+    -Intel Flash E28F004 (BIOS).
+  Video PCB:
+    -NEC 53G9037.
+    -IBM 20H2929.
+    -IBM 03H9515.
+    -Trident Cyber9385T.
+    -Two National Semiconductor DS90CF561MTD (LVDS 18-Bit Color Flat Panel Display [FPD] Link).
+    -Six KM416V256DT-L6 (256K x 16Bit CMOS Dynamic RAM with Fast Page Mode).
+    -AD722JR (RGB to NTSC/PAL Encoder).
+    -Philips SAA7110A WP.
```

Comment:
I'm curious, can you provide a PCB picture for the video block?
Cyber9385 is the very same chip used by a bunch of PC-9821 notebooks too, and there's a PCI card on VGA Legacy MK-3 that clearly shows a VGA BIOS, however I don't think there's one here unless integrated in the packaging?
![d2ba94f3222b824cf3568813b993f00a_XL](https://github.com/user-attachments/assets/e5e6ad8d-627f-453c-b876-0667346404ba)

Reply:
Yes!

![IBM Thinkpad 760XD Video card 46H5770 Back](https://github.com/user-attachments/assets/884a331b-d011-485e-89ec-b3dd97eb95e1)
![IBM Thinkpad 760XD Video card 46H5770 Front](https://github.com/user-attachments/assets/7a75bbc7-dc0d-4676-98af-c9e4f84e9dfc)